api/endpoints.py

```diff
@@ -202,14 +203,17 @@ def convert_audio():
 
     try:
         converted_output_path = converter.convert(output_format, **kwargs)
+    except AudioConversionError as error:
+        os.remove(output_path)
```

Comment:
os.remove could be in a finally block

Reply:
Thank you! I added it in the last commit